source/manual/alerts/slo-rate-checks-account.html.md

```diff
@@ -0,0 +1,44 @@
+---
+owner_slack: "#govuk-account-tech"
+title: GOV.UK Account SLO error rate exceeded 1% in 10 minute window
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+---
+
+The GOV.UK Account team has defined Service Level Objectives (SLOs) for
+reliability of the `account-api`.
+
+You can read more about these in [GOV.UK Accounts Service Level
+Objectives](https://docs.google.com/document/d/1HVXlyy2vdy9U0O-_7Nw8_Oso7VXQeKgJ2_-bi4pKlz0/edit)
+
+## Warning - "HTTP Error on GOV.UK Account has exceeded an acceptable threshold across the last 10 minutes"
+
+This warning triggers when over 1% of all HTTP status codes are deemed "bad
+responses" as a proportion of all responses within a rolling 10 minute time
+period.
+
+This is an early warning that, if these errors continue at a consistent rate,
+the GOV.UK Account team would spend its entire error budget of 1% across the
+full 4 week monitoring period.
+
+This warning automatically triggers a message to the `#govuk-accounts-tech`
+slack channel.
+
+No further action is required from GOV.UK 2ndline or any other team.
+
+## Warning - "Latency on GOV.UK Account has exceeded an acceptable threshold across the last 10 minutes"
+
+This warning triggers when over 1% of all HTTP responses are deemed
+"insufficiently quick" as a proportion of all responses within a rolling 10
```

Comment:
"too slow" is better than "insufficiently quick"

Reply:
good point @barrucadu, i thought "insufficiently quick" was the term used in Icinga. Whatever term we use in the docs should reflect what the user sees.